python/dgl/transforms/functional.py

```diff
@@ -3300,4 +3302,104 @@ def norm_by_dst(g, etype=None):
 
     return norm
 
+def rwpe(g, k):
+    r"""Random Walk Positional Encoding, as introduced in
+    `Graph Neural Networks with Learnable Structural and Positional Representations
+    <https://arxiv.org/abs/2110.07875>`__
+
+    This function computes the random walk positional encodings as landing probabilities
+    from 1-step to k-step, starting from each node to itself.
+
+    Parameters
+    ----------
+    g : DGLGraph
+        The homogeneous graph.
+    k : int
+        The number of random walk steps. The paper found the best value to be 16 and 20
+        for two experiments.
+
+    Returns
+    -------
+    Tensor
+        The random walk positional encodings.
+
+    Example
+    -------
+    >>> import dgl
+    >>> from dgl import functional as F
+    >>> g = dgl.graph(([0,1,1], [1,1,0]))
+    >>> F.rwpe(g, 2)
```

Comment:
```suggestion
    >>> dgl.rwpe(g, 2)
```

For consistency with previous functional transform practice.

Reply:
done